src/test/java/com/ericsson/ei/controller/TestQueryControllerImpl.java

```diff
@@ -48,27 +54,35 @@
 @ContextConfiguration(classes = App.class, loader = SpringBootContextLoader.class, initializers = TestContextInitializer.class)
 @RunWith(SpringJUnit4ClassRunner.class)
 @SpringBootTest(classes = { App.class })
-public class TestQueryControllerImpl extends ControllerTestBaseClass {
+@AutoConfigureMockMvc
+public class TestQueryControllerImpl {
```

Comment:
Please redo this test and reuse the functionality added in ControllerTestBaseClass, then this class does not need any MockMvc